runtime-checks/runtime-checks-store/src/main/java/com/duckduckgo/runtimechecks/store/RuntimeChecksRepository.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2023 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.runtimechecks.store
+
+import com.duckduckgo.app.global.DispatcherProvider
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+interface RuntimeChecksRepository {
+    fun updateAll(
+        runtimeChecksEntity: RuntimeChecksEntity,
+    )
+    var runtimeChecksEntity: RuntimeChecksEntity
```

Comment:
This should either be a `val` or a method. Otherwise one could just assign a value to it